doc/LLC/LLC_Quickstart_Tutorial.md

```diff
@@ -0,0 +1,181 @@
+
+Your `autorest.md` file should now look like:
+
+````md
+# `Azure.<your-sdk-name>` Code Generation
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+``` yaml
+input-file:
+- <swagger-file-link>
+namespace: <namespace>
+public-clients: true
+low-level-client: true
+security: AADToken
+security-scopes: <AAD-token-scope>
+```
+````
+
+**Learn more:** see the [.NET Guidelines for Namespaces](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-namespace-naming) to learn more about namespaces in the Azure SDK.
+
+#### 2. Run the Generate Command
+
+From a PowerShell command prompt, navigate to the directory holding `Azure.<your-sdk-name>.sln`. Run the following commands:
+
+```
+sdk\<your-service-name>\Azure.<your-sdk-name>\src> dotnet msbuild /t:GenerateCode
+```
+
+#### 3. Explore the Generated Code
+
+After you've run the `GenerateCode` command above, you should find a `Generated` folder in your `Azure.<your-sdk-name>` project.
+
+If you're interested, you can look through the swagger file to see how the autorest.csharp extension translated the REST API definition into C# code. This could help you later if you decide to make adjustments to your API design in the swagger file directly, so they can be used for all Tier 1 languages.
+
+**Learn more**: see the [OpenAPI Swagger Documentation](https://swagger.io/docs/) to learn more about swagger file formats.
+
+#### 4. Save and commit your changes to your branch
+
+This will make it easier to see the changes you make when you re-generate the code later.
+
+Congratulations, you've generated your first Azure SDK library! Next, we'll discuss how we can add tests, samples and documents.
+
+## Tests
+
+In this section, we will talk about adding unit tests and live tests and how to run them.
+
+- Add back `Azure.Template.LLC.Tests` csproj in your `Azure.<your-sdk-name>.sln`.
+- Rename the `Azure.Template.LLC.Tests` project to `Azure.<your-sdk-name>.Tests`.
+- Rename the `TemplateServiceTestEnvironment.cs` file to `<client-name>TestEnvironment.cs` and update the test environment. Before running live tests you will need to create live test resources, please refer [this](https://github.com/Azure/azure-sdk-for-net/blob/main/eng/common/TestResources/README.md) to learn more about how to manager resources and update test environment.
```

Comment:
Here you use `<client-name>` without explaining what it is our how to find it.  Can you add some explanation here?

Reply:
Sure! Thanks for the suggestion.